streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -735,7 +735,6 @@ public void shouldCommitNonCorruptedTasksOnTaskCorruptedException() {
         topologyBuilder.addSubscribedTopicsFromAssignment(anyObject(), anyString());
         expectLastCall().anyTimes();
         expectRestoreToBeCompleted(consumer, changeLogReader);
-        consumer.commitSync(eq(emptyMap()));
```

Comment:
These unit tests mocking are changed since originally we would send empty offsets.

Reply:
Since we won't commit empty offsets now, I think we should add test for this improvement. So, I'm thinking, instead of removing this line, we should verify that the `commitSync(emptyMap)` is not called during this test.

```java
consumer.commitSync(eq(emptyMap()));
expectLastCall().andStubThrow(new AssertionError("should not invoke commitSync when offset map is empty"));
```
What do you think?